src/powerlaces/boot_figreload.clj

```diff
@@ -90,7 +90,7 @@
   the fileset."
   [tmpfiles]
   (mapv #(-> {}
-             (assoc :build-id (util/build-id (:path %)))
+             (assoc :build-id (-> % b/tmp-file .getName util/build-id))
```

Comment:
Uhm, right, so there can be two `build-id` with the same name. This concept is a bit different from what figwheel considers a build-id I guess? I will need to calculate it in a different way then, maybe hashing the path?